example/spec/parallel/testAPI/collectionSpec.js

```diff
@@ -46,13 +52,56 @@ describe('Collections API', () => {
         bucket: config.bucket,
         key: recordCreatedKey,
       })).toBeResolved();
+      const savedEvent = await getJsonS3Object(config.bucket, recordCreatedKey);
+      const message = JSON.parse(savedEvent.Records[0].Sns.Message);
+      expect(message.event).toEqual('Create');
+      expect(omit(message.record, ['createdAt', 'updatedAt'])).toEqual(omit(collection, ['createdAt', 'updatedAt']));
+    }
+  });
+
+  it('updating a collection publishes a record to the collection reporting SNS topic', async () => {
+    if (beforeAllFailed) {
+      fail('beforeAll() failed');
+    } else {
+      const originalCollection = collection;
+      const updatedCollection = {
+        ...originalCollection,
+        reportToEms: false,
+      };
+      const originalCollectionFromApi = await getCollection({
```

Comment:
We should move at least this request into the above test for the creation SNS message so that we can compare against that rather than comparing against the `collection` fixture

Reply:
[`68b2fb6` (#2432)](https://github.com/nasa/cumulus/pull/2432/commits/68b2fb64fb360c103d4a24de28286ad68b4c75fb)